modules/concom/pages/panes.inc

```diff
@@ -105,7 +105,17 @@ function meetings()
                 if ($nowDate == $checkDate) {
                     echo "<td class='CONCOM-checkin-now'><a href='index.php?Function=concom/signin&meeting=".$value['MeetingID']."'>Sign In Now!</a>";
                 } else {
-                    echo "<td>Not Checked In";
+                    echo "<td>Not Checked In -";
+                    $checkPending = check_meeting_pastrequest($value['MeetingID']);
+
+                    if ($checkPending) {
+                        echo "Attendance request pending";
+                    } else {
+                      echo "<form action='index.php?Function=concom/pastcredit&meeting=".$value['MeetingID']."' method='post' name='butiwasthere' id='butiwasthere' class='UI-container'>";
+                      echo "    <input type='hidden' name='meeting' value='".$value['MeetingID']."'>";
+                      echo "    <button class='UI-OrangeButton'>But I attended!</button>";
```

Comment:
UI-OrangeButton is not defined anywhere as a style. you want "UI-orangebutton"

Reply:
I couldn't see it, thank you